sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_operations/_operations.py

```diff
@@ -0,0 +1,746 @@
+            raise HttpResponseError(response=response)
+
+        response_headers = {}
+        response_headers['location']=self._deserialize('str', response.headers.get('location'))
+        response_headers['schema-id']=self._deserialize('str', response.headers.get('schema-id'))
+        response_headers['schema-id-location']=self._deserialize('str', response.headers.get('schema-id-location'))
+        response_headers['schema-group-name']=self._deserialize('str', response.headers.get('schema-group-name'))
+        response_headers['schema-name']=self._deserialize('str', response.headers.get('schema-name'))
+        response_headers['schema-version']=self._deserialize('int', response.headers.get('schema-version'))
+
+        if _stream:
+            deserialized = response.iter_bytes()
+        else:
+            deserialized = _deserialize(
+                bytes,
+                response.json()
+            )
+
+        if cls:
+            return cls(pipeline_response, deserialized, response_headers) # type: ignore
+
+        return deserialized # type: ignore
+
+
+
+    @distributed_trace
+    def get_schema_id_by_content(  # pylint: disable=inconsistent-return-statements
+        self,
+        group_name: str,
+        name: str,
+        schema_content: bytes,
+        **kwargs: Any
+    ) -> None:
+        """Get ID for existing schema.
+
+        Gets the ID referencing an existing schema within the specified schema group, as matched by
+        schema content comparison.
+
+        :param group_name: Name of schema group. Required.
+        :type group_name: str
+        :param name: Name of schema. Required.
+        :type name: str
+        :param schema_content: String representation (UTF-8) of the registered schema. Required.
+        :type schema_content: bytes
+        :keyword content_type: The content type for given schema. Default value is "application/json;
+         serialization=Avro".
+        :paramtype content_type: str
+        :keyword bool stream: Whether to stream the response of this operation. Defaults to False. You
+         will have to context manage the returned stream.
+        :return: None
+        :rtype: None
+        :raises ~azure.core.exceptions.HttpResponseError:
+        """
+        error_map = {
+            401: ClientAuthenticationError, 404: ResourceNotFoundError, 409: ResourceExistsError, 304: ResourceNotModifiedError
+        }
+        error_map.update(kwargs.pop('error_map', {}) or {})
+
+        _headers = case_insensitive_dict(kwargs.pop("headers", {}) or {})
+        _params = kwargs.pop("params", {}) or {}
+
+        content_type: str = kwargs.pop('content_type', _headers.pop('content-type', "application/json; serialization=Avro"))
+        cls: ClsType[None] = kwargs.pop(
+            'cls', None
+        )
+
+        _content = json.dumps(schema_content, cls=AzureJSONEncoder)  # type: ignore
```

Comment:
different from current operation behavior.

Reply:
passing a string into this results in an escape `\` being added to all the quotations in the string, which changes the content.

Reply:
Izzy's working on a PR to fix this